test/CLISpec.hs

```diff
@@ -217,6 +217,31 @@ spec = do
               ]
           ]
     
+    it "saves steps to directory with --steps-dir" $ do
+      let stepsDir = "test-steps-temp"
+      -- Clean up if directory exists
+      dirExists <- doesDirectoryExist stepsDir
+      when dirExists $ removeDirectoryRecursive stepsDir
+      
+      -- Run the command with --steps-dir
+      withStdin "Q -> [[a -> 42]]" $ do
+        (_, _) <- withStdout (try (runCLI ["rewrite", "--normalize", "--steps-dir=" ++ stepsDir]) :: IO (Either ExitCode ()))
+        
+        -- Check that directory was created
+        dirCreated <- doesDirectoryExist stepsDir
+        dirCreated `shouldBe` True
+        
+        -- Check that at least one step file was created
+        files <- listDirectory stepsDir
+        length files `shouldSatisfy` (>= 1)
+        
```

Comment:
@yarkivaev we're trying not to leave empty lines inside function body: https://www.yegor256.com/2014/11/03/empty-line-code-smell.html